IFC4x3/Properties/r/RMSWithstandVoltage_29RnvrWq5DXAWWDKKhvsNo/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RMSWithstandVoltage_29RnvrWq5DXAWWDKKhvsNo" Name="RMSWithstandVoltage" UniqueId="896f1e75-8341-4d84-a820-35452be765f2" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
Still would prefer `MaximimumRMSVoltage`

Reply:
I think the "withstand voltage" is a standard name. @fhulin , could you check IEC standard?